scripts/local_k8s/config.py

```
@@ -1,53 +1,140 @@
         )
+
+
+def _container_engine_from_env() -> ContainerEngine:
+    """Return the validated container engine environment value."""
+    raw_value = os.environ.get("WILDSIDE_CONTAINER_ENGINE", DEFAULT_CONTAINER_ENGINE)
+    if raw_value in ("docker", "podman"):
+        return raw_value
+    raise LocalK8sError(
+        "WILDSIDE_CONTAINER_ENGINE must be one of docker, podman; "
+        f"got {raw_value!r}"
+    )
```

Ruff (0.15.20)

[warning] Avoid specifying long messages outside the exception class

(TRY003)